portal-web/test/functional/com/liferay/portalweb/paths/analyticscloud/ACTimeFilter.path

```diff
@@ -0,0 +1,60 @@
+<html>
+<head>
+  <title>ACTimeFilter</title>
+</head>
+<body>
+<table cellpadding="1" cellspacing="1" border="1">
+  <thead>
+  <tr><td rowspan="1" colspan="3">ACTimeFilter</td></tr>
+  </thead>
+  <tbody>
+  <!-- TIME FILTER ELEMENTS  -->
+  <tr>
+    <td>TIME_FILTER_DROPDOWN</td>
+    <td>//div[@class='dropdown dropdown-range-key-root']/button</td>
```

Comment:
Hey @ctang3 , just a suggestion in order to let your selector more robust, I would build it using a `contains` pointing to one of the multiple classes available, so if a class is added/removed, your selector will have less chance of being impacted by these changes.

So the selector would be something like:

`//div[contains(@class,'dropdown-range-key-root')]/button`